src/NuGet.Core/NuGet.PackageManagement/NuGet.PackageManagement.csproj

```diff
@@ -35,7 +35,7 @@
     <Reference Include="System.Net.Http" />
   </ItemGroup>
 
-  <ItemGroup Condition=" '$(TargetFramework)' == '$(NetStandardVersion)' or '$(TargetFramework)' == 'net6.0' ">
+  <ItemGroup Condition=" '$(TargetFramework)' == '$(NetStandardVersion)' or '$(TargetFramework)' == 'net7.0' ">
```

Comment:
Just curious, why does this assembly is needed for source-build? By looking at the description, it says this assembly is for Visual Studio.

Reply:
We still build NuGet.PackageManagement during source-build. We prefer to have the repo builds and the source-build be as similar as possible, so we usually only exclude things if they include pre-built binaries or have licensing issues.

As for this specific reference to System.ComponentModel.Compisition, NuGet.PackageManagement doesn't build without this reference. it looks like it's shipped in the SDK, so there's no issue referencing it here.